packages/react-native-host/cocoa/RNXBridgelessHeaders.h

```diff
@@ -8,8 +8,10 @@
 
 #if USE_HERMES
 #import <ReactCommon/RCTHermesInstance.h>
+#elif USE_V8
+#import <v8runtime/V8ExecutorFactory.h>
 #else
-#import <ReactCommon/RCTJscInstance.h>
+#import <React-jsc/RCTJscInstance.h>
```

Comment:
We want another check here to maintain backwards compatibility:
```suggestion
#elif __has_include(<React-jsc/RCTJscInstance.h>)
#import <React-jsc/RCTJscInstance.h>
#else
#import <ReactCommon/RCTJscInstance.h>
```